src/components/common/Header.jsx

```diff
@@ -23,29 +23,26 @@ function Header() {
     { 'Home': isHomePage },
   );
 
-  const generalNavItems = (
+  const navItems = (
     <Nav>
-      <Nav.Item><Nav.Link href="/">Home</Nav.Link></Nav.Item>
-      {/* <Nav.Item><Nav.Link href="/exploretechla2021">Event Info</Nav.Link></Nav.Item> */}
-      <Nav.Item><Nav.Link href={`/${HomePageSections.TEAM.name}`}>Our Team</Nav.Link></Nav.Item>
-      <Nav.Item><Nav.Link href="/resources">exploretech.la 2021</Nav.Link></Nav.Item>
-    </Nav>
-  );
 
-  const homeNavItems = (
-    <Nav>
-      <Nav.Item><Nav.Link href={`#${HomePageSections.ABOUT.name}`}>About</Nav.Link></Nav.Item>
-      {/* <Nav.Item><Nav.Link href={`/exploretechla2021`}>Event Info</Nav.Link></Nav.Item> */}
-      <Nav.Item><Nav.Link href={`#${HomePageSections.GET_INVOLVED.name}`}>Get Involved</Nav.Link></Nav.Item>
-      <Nav.Item><Nav.Link href={`#${HomePageSections.SPEAKERS.name}`}>Speakers</Nav.Link></Nav.Item>
-      <Nav.Item><Nav.Link href={`#${HomePageSections.SPONSORS.name}`}>Sponsors</Nav.Link></Nav.Item>
-      <Nav.Item><Nav.Link href={`/${HomePageSections.TEAM.name}`}>Our Team</Nav.Link></Nav.Item>
-      <Nav.Item><Nav.Link href={`/resources`}>exploretech.la 2021</Nav.Link></Nav.Item>
+      <NavDropdown title="Home" id="navbarScrollingDropdown" href="/">
+          <NavDropdown.Item href="/">Home</NavDropdown.Item>
+          <NavDropdown.Item href={`/#${HomePageSections.ABOUT.name}`}>About</NavDropdown.Item>
+          <NavDropdown.Item href={`/#${HomePageSections.GET_INVOLVED.name}`}> Get Involved </NavDropdown.Item>         
```

Comment:
i might have mentioned this a while back, but a very powerful formatting tool developers use is called `prettier`! there's a way to configure prettier so that it runs automatically on VSCode on save (which i could try to look up for you) but for now before you push up code do you mind running `prettier --write`